vscodium: 1.74.3.23010 -> 1.75.0.23033

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from passthru.updateScript.


meta.description for vscodium is: Open source source code editor developed by Microsoft for Windows,
Linux and macOS (VS Code without MS branding/telemetry/licensing)

meta.homepage for vscodium is: https://github.com/VSCodium/vscodium




###### Updates performed
- Ran passthru.UpdateScript

###### To inspect upstream changes

- [Release on GitHub](https://github.com/VSCodium/vscodium/releases/tag/1.75.0.23033)

- [Compare changes on GitHub](https://github.com/VSCodium/vscodium/compare/1.74.3.23010...1.75.0.23033)

###### Impact

<details>
<summary>
<b>Checks done</b> (click to expand)
</summary>

---

- built on NixOS
- Warning: a test defined in `passthru.tests` did not pass
- found 1.75.0.23033 with grep in /nix/store/afs67d584pr8df8pgkhldmgbymafzmwp-vscodium-1.75.0.23033
- found 1.75.0.23033 in filename of file in /nix/store/afs67d584pr8df8pgkhldmgbymafzmwp-vscodium-1.75.0.23033

---

</details>
<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
2 total rebuild path(s)

2 package rebuild(s)

First fifty rebuilds by attrpath
vscodium
vscodium-fhs
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/afs67d584pr8df8pgkhldmgbymafzmwp-vscodium-1.75.0.23033 \
  --option binary-caches 'https://cache.nixos.org/ https://nix-community.cachix.org/' \
  --option trusted-public-keys '
  nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A vscodium https://github.com/r-ryantm/nixpkgs/archive/535b05bc2dcce4368d8217eb62f13e038a8a10c6.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/afs67d584pr8df8pgkhldmgbymafzmwp-vscodium-1.75.0.23033
ls -la /nix/store/afs67d584pr8df8pgkhldmgbymafzmwp-vscodium-1.75.0.23033/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>vscodium</li>
    <li>vscodium-fhs</li>
  </ul>
</details>

---

###### Maintainer pings

cc @Synthetica9 @turion @bobby285271 for [testing](https://github.com/ryantm/nixpkgs-update/blob/master/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

r-ryantm committed:
vscodium: 1.74.3.23010 -> 1.75.0.23033

bobby285271 commented:
@ofborg test vscodium.xorg

bobby285271 commented:
:sweat_smile: The test failure is not that interesting

- https://github.com/microsoft/vscode/pull/168231

bobby285271 committed:
nixosTests.vscodium: rename "Get Started" to "Welcome"

https://github.com/microsoft/vscode/commit/c258547b50e70486896a3dafcc1447fe28ea4400

bobby285271 commented:
@ofborg test vscodium.xorg

bobby285271 commented:
Sorry I am doing bad QA this time

- #214666
